tutorials/evals/evals_sql_correctness_eval_with_custom_agent.ipynb

```diff
@@ -0,0 +1,2574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !curl "https://www.sqlitetutorial.net/wp-content/uploads/2018/03/chinook.zip" -O ./chinook.zip
</pre><p>We can avoid saving a file to disk by piping the output of curl to unzip instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2802/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>